### PR TITLE
[REEF-1163] Replace Apache feather logo on website with 2016 version

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
JIRA:
  [REEF-1163](https://issues.apache.org/jira/browse/REEF-1163)

This closes #

### CODE REVIEW

File: website/src/site/site.xml [hunk 1]
```diff
@@ -51,7 +51,7 @@ under the License.
     <href></href>
   </bannerLeft>
   <bannerRight>
-    <src>apache-feather.gif</src>
+    <src>apache-feather.png</src>
```
Comment (markusweimer) [inline website/src/site/site.xml]: Is there a way for us to reference the file from apache.org directly?

Comment (dongjoon-hyun) [inline website/src/site/site.xml]: Unfortunately, the image size is huge. So, it breaks out website layout.

Comment (dongjoon-hyun) [inline website/src/site/site.xml]: This is the original ones.
http://apache.org/img/asf_logo.png  (Full logo)
http://apache.org/img/feather_glyph_notm.png  (Feather logo)

Comment (markusweimer) [inline website/src/site/site.xml]: Ah, we'll use our own then :)

Comment (dongjoon-hyun) [inline website/src/site/site.xml]: @markusweimer , could you merge this, too? :)

Comment (markusweimer) [inline website/src/site/site.xml]: Uh oh, I will have to setup myself for website merges again. Any other volunteers?

Comment (dongjoon-hyun) [inline website/src/site/site.xml]: I see. I think merging-to-master (except publishing) is enough at this time since we will publish 0.14 soon.

Comment (markusweimer) [inline website/src/site/site.xml]: I'll do that. Feel free to push to the website.

Comment (dongjoon-hyun) [inline website/src/site/site.xml]: Thank you!

**Pull request closed without merging**